app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -33,35 +33,36 @@ def supports_authentication?(authtype)
     supported_auth_types.include?(authtype.to_s)
   end
 
-  def self.raw_connect(server, port, username, password, service = "Service")
+  def self.raw_connect_metrics(connect_options)
+    require 'ovirt_metrics'
+    OvirtMetrics.connect(connect_options)
+  end
+
+  def self.raw_connect(connect_options)
```

Comment:
[Same thing](https://github.com/ManageIQ/manageiq/pull/3811/files#r37029933) here.  Put the explicit parameter list back.